.pre-commit-config.yaml

```diff
@@ -0,0 +1,62 @@
+fail_fast: true
+
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v5.0.0
+  hooks:
```

Comment:
Eventually (but with the real meaning of "eventually in the far future"), we may want to add `mypy` as a hook to have "typed" code. We can't do it right now as it will be overwhelming do all the required changes, but it's something to take in consideration for the future!

Reply:
created an issue to not forget about